.github/dependabot.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    allow:
+      - dependency-type: "production"
+    target-branch: "main"
+    labels:
+      - "npm dependencies"
+      - "frontend"
+    open-pull-requests-limit: 3
```

Comment:
I think we can probably remove this limit ?

Reply:
removing the limit as suggested. We can add it back in later incase we feel like dependabot is introducing a lot of noise with PRs per week 👍